src/Common/src/CoreLib/System/String.cs

```diff
@@ -194,7 +194,12 @@ private unsafe string Ctor(sbyte* value, int startIndex, int length)
                 throw new ArgumentOutOfRangeException(nameof(length), SR.ArgumentOutOfRange_NegativeLength);
```

Comment:
You can also delete the theoretically incorrect ifdef in `Ctor(sbyte* value)` while you are on it as we have discussed over email.